sources/core/Stride.Core.Reflection/TypeDescriptorFactory.cs

```diff
@@ -106,7 +109,7 @@ protected virtual ITypeDescriptor Create(Type type)
             }
             else if (type.IsArray)
             {
-                if (type.GetArrayRank() == 1 && !type.GetElementType().IsArray)
+                if (type.GetArrayRank() == 1 && !type.GetElementType()!.IsArray)
```

Comment:
Looks great but I noticed you did used null-forgiving operator for Nullable type that might be null. Is it intentional @IXLLEGACYIXL ?

Reply:
This shouldn't return null when the type is an array, which it is given the if test above :)

Reply:
Right, makes sense! 😅